component/web/controller/src/main/java/org/exoplatform/web/WebAppController.java

```diff
@@ -332,8 +335,13 @@ public void service(HttpServletRequest req, HttpServletResponse res) throws Exce
                                 if (debug) {
                                     log.debug("Starting RequestLifeCycle for handler " + handler);
                                 }
-                                RequestLifeCycle.begin(ExoContainerContext.getCurrentContainer());
                                 started = true;
+                                ExoContainer container = ExoContainerContext.getCurrentContainer();
+                                if (!(container instanceof PortalContainer)) {
+                                  //We need this log to trace the issue for TA-423 if the bug still happen
+                                  log.error("current container is " + container + ", it should be a PortalContainer instance to start a portal request lifecycle");
```

Comment:
- Hi Thomas, thanks for you comment, as you know, the PR help to fix the potential issue, i mean there may be other bug that can lead to "Session is closed!" exception, but we not sure, and this exception is hard to produced on local computer, that's why i put that log, then IF we still see this exception, we can trace it in the log of Community project (I guess the possible level here can be warning or error, not debug)

- Do you think it's better to be warning instead of error log level ?